docs/pages/getting-started/installation.mdx

```diff
@@ -141,6 +141,8 @@ app.set("trust proxy", true)
 app.use("/auth/*", ExpressAuth({ providers: [] }))
 ```
 
+Note this creates the auth API, but does not yet protect resources.
```

Comment:
Small copy nit, and let's link them to what they would need going forward from here 

```suggestion
Note this creates the Auth.js API, but does not yet protect resources. Continue on to [protecting resources](/getting-started/session-management/protecting) for more details. 
```